README.md

```diff
@@ -24,6 +24,7 @@
 
 [![GitHub Discussions](https://img.shields.io/static/v1?label=Discussions&message=Ask&color=blue&logo=github)](https://github.com/orgs/glass-dev/discussions)
 [![Slack](https://img.shields.io/badge/join-Slack-4A154B)](https://glass-dev.github.io/slack)
+[![All Contributors](https://img.shields.io/github/all-contributors/glass-dev/glass?color=ee8449&style=flat-square)](#contributors)
```

Comment:
Do we also want the badge? See here for an example https://github.com/all-contributors/all-contributors/blob/master/README.md.

Reply:
This would be nice to have.